article/tests/test_helpers.py

```diff
@@ -40,8 +40,8 @@ def test_database_create_article_read_calls_api(
 ):
     mock_bulk_create_article_read.return_value = create_response(200)
 
-    manager = helpers.DatabaseArticlesReadManager(sso_request)
-    manager.bulk_persist_article(article_uuids=['123'])
+    manager = helpers.DatabaseArticlesViewedManager(sso_request)
+    manager.persist_articles(article_uuids=['123'])
```

Comment:
`save_article_views` or `persist_article_views` would be a better name, as it's the views, not the articles being persisted here.